intrinsics:
  camera_matrix:
  - 376.6869473024056
  - 0.0
  - 318.1658197521198
  - 0.0
  - 376.67926795109804
  - 247.68392644478962
  - 0.0
  - 0.0
  - 1.0
  dist_coeffs:
  - -0.007359038257331093
  - -0.10004233758035878
  - -0.00230046817365035
  - -0.0008217400189285176
  - 0.3129295580544872
  image_size:
  - 640
  - 480
  reprojection_error: 0.0
extrinsics:
  rvec:
  - -1.5130584605641855
  - -2.0014952898403853
  - -1.6813452928509658
  tvec:
  - 0.45407239944857686
  - 0.411261138807873
  - 0.6100738220517314
  T_cam_robot:
  - -0.49257181794356697
  - 0.7288678429071954
  - 0.47552567937345047
  - 0.45407239944857686
  - 0.5941634190122907
  - -0.11759154329224752
  - 0.7957022435897567
  - 0.411261138807873
  - 0.6358795763943003
  - 0.674480464151453
  - -0.3751443292956004
  - 0.6100738220517314
  - 0.0
  - 0.0
  - 0.0
  - 1.0
